docs/api/php_api/php_api.md

```diff
@@ -5,7 +5,7 @@ page_type: reference
 
 # PHP API
 
-The [public PHP API](../php_api_reference/) enables you to interact with [[= product_name =]]'s Repository and content model from your PHP code.
+The [public PHP API](/api/php_api/php_api_reference/index.html) enables you to interact with [[= product_name =]]'s Repository and content model from your PHP code.
```

Comment:
* Fix `INFO    -  Doc file 'api/php_api/php_api.md' contains an unrecognized relative link '../php_api_reference/', it was left as is.`
* Use `absolute_links: relative_to_docs `